frontend/src/main/scala/akkaviz/frontend/components/ReplTerminal.scala

```diff
@@ -68,14 +86,28 @@ class ReplTerminal extends Component {
     ws = _ws
   }
 
-  private def setupReplTerminal(element: Element): Unit = {
-    setupWebsocket()
+  private[this] def setupReplTerminal(element: Element): Unit = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/lustefaniak/akka-viz/file/2167023955/issues/source?bid=3121282&fileBranchId=3161575#l89)